pkg/snapshot/restorer/restorer.go

```diff
@@ -659,7 +662,20 @@ func applyEventsToEtcd(client *clientv3.Client, events []event) error {
 			return fmt.Errorf("Unexpected event type")
 		}
 	}
-	_, err := client.Txn(ctx).Then(ops...).Commit()
+
+	// Divide the total operations in smaller chunks as per MaxTxnOps
+	fmt.Println("Length of ops is: ", len(ops))
+	var err error
+	/*for _, op := range ops {
+		_, err = client.Do(context.Background(), op)
+	}*/
+	for len(ops) > int(ro.Config.MaxTxnOps) {
+		chunk := ops[ro.Config.MaxTxnOps:]
```

Comment:
Shouldn't the chunks be applied left (older) to right (newer)?
```suggestion
		chunk := ops[:ro.Config.MaxTxnOps]
```